config/sync/views.view.content_paragraphs.yml

```diff
@@ -0,0 +1,340 @@
+uuid: 66163bca-af6b-4db4-b820-3579418763ea
```

Comment:
This config is 99% original, except I've added:

- A new display, for showing sliders
- Removed the filter that says content must be articles
- Added content type as an contextual filter